prow/plank/controller.go

```diff
@@ -591,4 +607,123 @@ func (c *Controller) patchProwjob(srcPJ prowapi.ProwJob, destPJ prowapi.ProwJob)
 	_, err = c.prowJobClient.Patch(srcPJ.Name, ktypes.MergePatchType, patch)
 	c.log.WithFields(pjutil.ProwJobFields(&destPJ)).Debug("Patched ProwJob.")
 	return err
+
+}
+
+type plankReconciliationMetrics struct {
```

Comment:
Can we remove the metrics we move over to plank from sinker?

Also the `prowJobsXXX` metrics don't make sense here, as the ProwJob cleanup still remains in sinker